lib/bolt/bolt_option_parser.rb

```diff
@@ -298,15 +296,24 @@ def initialize(options)
       update
```

Comment:
Is this necessary in both the initialize method and at https://github.com/puppetlabs/bolt/blob/dfe4f2916b7f0952aee6bfb2c9750ed8787a2d4e/lib/bolt/cli.rb#L64 ?

Reply:
Removing either instance of `update` results in some failed tests. `parser.update` causes several to fail while the one in the initialize method only causes `./spec/bolt/cli_spec.rb:101` to fail.